packages/cli/package.json

```diff
@@ -34,9 +34,9 @@
   "homepage": "https://github.com/preactjs/preact-cli",
   "devDependencies": {
     "@types/express": "^4.17.13",
-    "@types/jest": "^27.4.0",
+    "@types/jest": "^24.9.1",
     "html-looks-like": "^1.0.2",
-    "jest": "^27.0.1",
+    "jest": "^24.9.0",
```

Comment:
Chased down the [weird config file hack](https://github.com/preactjs/preact-cli/blob/aea6fedbc0fa363f94296c5c3912c349f5312527/packages/cli/lib/lib/webpack/transform-config.js#L110-L116) I introduced in #1623, turns out to be an issue in `esm` (the package, not the format) in Jest > v25.

Issue filed at https://github.com/facebook/jest/issues/12493, but I needed to use ESM within Jest and that hack only worked for CJS config files in our test suite. Downgrading shouldn't be problematic, so I went this route.